imod/msw/copy_files.py

```diff
@@ -0,0 +1,55 @@
+from pathlib import Path
+from shutil import copy2
+from typing import cast
+
+import numpy as np
+import xarray as xr
+
+from imod.logging import logger
+from imod.logging.loglevel import LogLevel
+from imod.msw.pkgbase import MetaSwapPackage
+from imod.typing import Imod5DataDict
+
+_LOG_MESSAGE_TEMPLATE = """\
+Will not copy files {filtered}, these will be generated by iMOD Python
+instead."""
+
+
+class CopyFiles(MetaSwapPackage):
```

Comment:
A class name should be a Noun and its method should be verbs.
Consider naming this class FileCopier

Reply:
Done